### ISSUE TITLE
Root package-lock.json is not included in vsix archive

### ISSUE THREAD

**Issue by alexfedoseev** [rescript-lang/rescript-vscode#233]
It causes issues on installation of Atom IDE package. I had to rebuild and vendor it. Is there a specific reason not including it in the vsix (while server’s lock file is included)?

**Comment by chenglou** [comment]
Hey. We didn't specify such thing, so I'm guessing that's the default `.vscodeignore` behavior?

More importantly though, are you able to only vendor the `server/` folder? The `client/` folder and everything above isn't useful to you right?

**Comment by alexfedoseev** [comment]
@chenglou Yeah, I tried to vendor server only initially, but the problem is that server's code uses some modules from the parent node_modules so I had to vendor both package.json's.

**Comment by chenglou** [comment]
This is fixed in master. I'll publish a few version later and close this.
